dot/state/grandpa.go

```diff
@@ -264,3 +284,76 @@ func (s *GrandpaState) GetNextResume() (*big.Int, error) {
 
 	return big.NewInt(0).SetBytes(num), nil
 }
+
+var (
+	prevotesPrefix   = []byte("pv")
```

Comment:
const as string?

Reply:
or just define them in `prevotesKey` and `precommitsKey`

Reply:
updated